app/Http/Controllers/UsersController.php

```diff
@@ -67,12 +67,10 @@ public function search(Request $request)
         // Attempt to fetch all users.
         $users = $this->registrar->search($query);
 
-        if (! $users) {
```

Comment:
`! $users` should never be the case because if there are not results, `$users` looks like this: 
```
NorthstarUserCollection {#424 ▼
  #items: []
  #total: 0
  #perPage: 20
  #currentPage: 1
  #hasMore: null
  #paginator: null
}
```